packages/flutter/lib/src/material/app_bar_theme.dart

```diff
@@ -0,0 +1,135 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'text_theme.dart';
+import 'theme.dart';
+
+/// Defines default property values for descendant [AppBar] widgets.
+///
+/// Descendant widgets obtain the current [AppBarTheme] object using
+/// `AppBarTheme.of(context)`. Instances of [AppBarTheme] can be customized
+/// with [AppBarTheme.copyWith].
+///
+/// Typically an [AppBarTheme] is specified as part of the overall [Theme] with
+/// [ThemeData.appBarTheme].
+///
+/// All [AppBarTheme] properties are `null` by default. When null, the [AppBar]
+/// constructor provides defaults.
```

Comment:
The constructor doesn't supply any anymore.

Reply:
Think it takes values from ThemeData mostly.

Reply:
Good point Will, probably just remove the word "constructor"

`When null, the [AppBar] provides defaults.`

Reply:
Updated